homeassistant/components/fritzbox/sensor.py

```diff
@@ -96,7 +96,9 @@ class FritzSensorEntityDescription(
         device_class=SensorDeviceClass.VOLTAGE,
         state_class=SensorStateClass.MEASUREMENT,
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
-        native_value=lambda device: device.voltage / 1000 if device.voltage else 0.0,
+        native_value=lambda device: device.voltage / 1000
+        if getattr(device, "voltage", None)
```

Comment:
Should you check explicitely for `None` to guard for `device.voltage` being 0?

Reply:
if there is a value, it must be >0, since the smart plug needs some voltage to work 😄

Reply:
regarding style, I think we try to avoid multi-line lambdas but if this is a hotfix maybe it doesn't apply here